wrta:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  horizontal_panning_controller:
    type: position_controllers/JointPositionController
    joint: horizontal_panning
    pid: {p: 100.0, i: 0.01, d: 10.0}
  vertical_pitching_controller:
    type: position_controllers/JointPositionController
    joint: vertical_pitching
    pid: {p: 100.0, i: 0.01, d: 10.0}
  length_extension_controller:
    type: position_controllers/JointPositionController
    joint: length_extension
    pid: {p: 100.0, i: 0.01, d: 10.0}
  wrist_rotation_controller:
    type: position_controllers/JointPositionController
    joint: wrist_rotation
    pid: {p: 100.0, i: 0.01, d: 10.0}
  wrist_pitching_controller:
    type: position_controllers/JointPositionController
    joint: wrist_pitching
    pid: {p: 100.0, i: 0.01, d: 10.0}
